easybuild/easyconfigs/r/Ruby/Ruby-2.7.1-GCCcore-8.3.0.eb

```
@@ -0,0 +1,148 @@
+name = 'Ruby'
+version = '2.7.1'
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with
+ a focus on simplicity and productivity. It has an elegant syntax that is
+ natural to read and easy to write."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['https://cache.ruby-lang.org/pub/ruby/%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['d418483bdd0000576c1370571121a6eb24582116db0b7bb2005e90e250eae418']
+
+builddependencies = [('binutils', '2.32')]
+
+exts_default_options = {
+    'source_urls': ['https://rubygems.org/downloads/'],
+    'source_tmpl': '%(name)s-%(version)s.gem',
+}
+
+exts_list = [
+    ('ffi', '1.12.2', {
+        'checksums': ['048ad01d5369f67075f943c16f1058f10663af2a66eedb87d921316ba1828e82'],
+    }),
+    ('childprocess', '3.0.0', {
+        'checksums': ['4579a87cdc962de252eebf1482a4185fad383ae7dbe29a746ba2be8e261280c5'],
+    }),
+    ('json', '2.2.0', {
+        'checksums': ['9dd1437156773f72c096058ec837faac1b00077121a3fd574e68f895ea3aa96b'],
+    }),
+    ('cabin', '0.9.0', {
+        'checksums': ['91c5394289e993e7037a6c869e3f212f31a5984d2b1811ac934f591c87446b2c'],
+    }),
+    ('backports', '3.17.0', {
+        'checksums': ['bb18a4c7a2a13828d18e348ea81183554adcaac4fc9db0ecd1f3d1dfbd7fdc8f'],
+    }),
+    ('arr-pm', '0.0.10', {
+        'checksums': ['0fa2d425a84eeeaf8f95268568e826723b41aeefa14511d1f0ed1368c30bdd1f'],
+    }),
+    ('clamp', '1.3.1', {
+        'checksums': ['0975e919dc76cfb1a63264fec5c2b97c4bba54cbdec7894ab9e77f0e87436a36'],
+    }),
+    ('multipart-post', '2.1.1', {
+        'checksums': ['d2dd7aa957650e0d99e0513cd388401b069f09528441b87d884609c8e94ffcfd'],
+    }),
+    ('multipart-post', '1.2.0', {
+        'checksums': ['ccf26a61706f128a2a3f3745aba0f79fde7b6fee068e03490eb189c2aaa5e78a'],
+    }),
+    ('faraday', '1.0.1', {
+        'checksums': ['381aee04fcc9effbe5fa7cc703d8f5f20293722f987ded4f958f77514cd29373'],
+    }),
+    ('faraday', '0.8.11', {
+        'checksums': ['b16608080bd0874d44b15db85833a75cc6dfcc7e7ffc5e0603d043c559f9bb94'],
+    }),
+    ('faraday_middleware', '1.0.0', {
+        'checksums': ['19e808539681bbf2e65df30dfbe27bb402bde916a1dceb4c7496dbe8de14334a'],
+    }),
```

Version 1.0.0 is too high, but 0.14.0 is quite recent and currently the highest supported by `gh`
```suggestion
    ('faraday_middleware', '0.14.0', {
        'checksums': ['4cb37ddd656b2c4de0bd684b72b08c34486f70560c31cb303cd506faef7ef2f4'],
    }),
```